webapp/src/main/java/com/box/l10n/mojito/react/ReactAppConfig.java

```diff
@@ -1,11 +1,13 @@
 package com.box.l10n.mojito.react;
 
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
 public class ReactAppConfig {
 
+    @JsonUnwrapped
```

Comment:
@magezil remove a level of "link" with the `@JsonUnwrapped` annotation. I didn't make sense to have two, I think it was just there as a result have having the map in an object

Reply:
Awesome! I agree that it doesn't make sense to have two.